client/src/components/Category/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./style.css";
+import axios from "axios";
+class Category extends React.Component {
+  state = {
+    result: []
+  };
+  componentDidMount() {
+    axios
+      .get("/category")
+      .then(data => {
+        return data;
+      })
+      .then(data => {
+        const resultArray = data.data.data.map(e => e);
```

Comment:
what `data.data.data` ?! make the names more clear

Reply:
i delete it and change to a simple name